CodePush.js

```diff
@@ -418,15 +418,15 @@ if (NativeCodePush) {
       ON_NEXT_RESUME: NativeCodePush.codePushInstallModeOnNextResume // Restart the app the next time it is resumed from the background
     },
     SyncStatus: {
-      CHECKING_FOR_UPDATE: 0,
-      AWAITING_USER_ACTION: 1,
-      DOWNLOADING_PACKAGE: 2,
-      INSTALLING_UPDATE: 3,
-      UP_TO_DATE: 4, // The running app is up-to-date
-      UPDATE_IGNORED: 5, // The app had an optional update and the end-user chose to ignore it
-      UPDATE_INSTALLED: 6, // The app had an optional/mandatory update that was successfully downloaded and is about to be installed.
-      SYNC_IN_PROGRESS: 7, // There is an ongoing "sync" operation in progress.
-      UNKNOWN_ERROR: -1
+      UP_TO_DATE: 0, // The running app is up-to-date
```

Comment:
This was done so that the Cordova plugin and this plugin have identical numberings.